README.md

```diff
@@ -1,70 +1,152 @@
+
+To add a run to an existing workflow, specify the workflow's id. For a workflow with an id of 1
+```
+save.workflow=1 
+```
+
+#### save.title
+
+This is an optional field to set or update the title of a workflow.
+```
+save.title=This is an example title for a workflow
+```
+
+
+#### save.description
+
+This is an optional field to set or update the description of a workflow.
+```
+save.description=This is an example description for a workflow
+```
+
+#### save.tags
+
+This is a comma delimited list of tags you want to set or update the tags of a workflow.
+```
+save.tags=fun,science,bill-nye
+```
+
+#### Some things to note
+
+YesWorkflow save calls all subsequent commands (extract, model, graph, recon). When calling graph, by default, there will be a dump of a graphviz string to terminal output. To hide this output when calling YesWorkflow save, it's recommended that you use the following configuration
+```
+graph.dotfile=out.txt
+```
+
+This puts the graphviz string into a file so you can see 'save' related content.
+
+
+## Website Navigation
+YesWorkflow Web Components has three major pages. The Home page, which contains a list of all workflows uploaded to the website. To view only the workflows associated with the currently logged in user, click the My Workflows tab. 
+
+![alt text](/demos/images/my_workflows.png "My Workflows")
+
+
+To navigate back to the homepage, click the YesWorkflow logo in the top left corner.
+
+On the Home page and My Workflows page one can enter a query into the search bar and filter workflows that are relevant to the search entry.
+
+By clicking on a workflow found on either the Home page or the My Workflows page you will be navigated to a more detailed view of the workflow. 
+![alt text](/demos/images/workflow_click.png "Click Here")
+
+The next page you will see is the detailed workflow page. If you are the owner of this workflow you will have extra privileges, such as editing and deleting the workflow.
 
-### Building a jar file
+You can select different versions of the workflow with the version dropdown. 
 
-You can build a new jar file from the above repository by following these steps:
+To navigate to a more detailed view of a run, click on one of the runs found in the list of runs. This run page will contain more details about the files associated with the run.
 
-1. Set up a development environment for yw-prototypes [Setting up YesWorkflow dev Environment](https://github.com/aniehuser/senior-design-group10#setting-up-yesworkflow-dev-environment)
-2. Open up a command prompt and navigate to the directory you cloned yw-prototypes into.
-3. Run `mvn package`
-    1. If you run into issues with tests from YwClient failing, you likely are running an old version YesWorkflow WebComponents on `localhost:8000`. You can terminate this process and the tests will be ignored.
-4. The new jar will be placed in the path `<path-to-yw-prototypes>/yw-prototypes/out/artifacts` folder.
```

Comment:
lets not remove instructions to build the jar file, maybe move it somewhere else?

Reply:
Thoughts on just referring to the other repo? You have to clone it to build the jar anyway

Reply:
perhaps, but this information isn't in the other repo. I wrote it up for gotcha's in setting up the dev environment